pkg/policyGovernance/security/scanTool/wire_scanTool.go

```diff
@@ -0,0 +1,8 @@
+package scanTool
+
+import "github.com/google/wire"
+
+var ScanToolWireSet = wire.NewSet(
+	NewScanToolMetadataServiceImpl,
```

Comment:
ea mode is direclty binding itself, should use this wireset there too

Reply:
done